code/gradle.properties

```diff
@@ -24,5 +24,5 @@ mavenRepoDescription=Android Places Extension for Adobe Mobile Marketing
 mavenUploadDryRunFlag=false
 
 # production versions for production build
-mavenCoreVersion=3.0.0-beta.1
+mavenCoreVersion=3.0.0-beta.1-SNAPSHOT
```

Comment:
Since you switched this to add `SNAPSHOT` suffix, you will need to change this to
https://github.com/adobe/aepsdk-places-android/blob/dev-v3.0.0/code/places/build.gradle.kts#L30 
`implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion")`